lib/foreman_tasks/dynflow/configuration.rb

```diff
@@ -59,6 +59,8 @@ def run_on_init_hooks(world)
 
     def initialize_world(world_class = ::Dynflow::World)
       world_class.new(world_config)
+    rescue ArgumentError
```

Comment:
Because `DummyWorld` (the one that is used in tests) have zero arguments in its `initialize` method: https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/testing/dummy_world.rb#L10